developer-docs-site/docs/tutorials/run-a-fullnode-on-gcp.md

```diff
@@ -1,10 +1,13 @@
 ---
-title: "Run a FullNode on GCP"
+title: "Single Doc for GCP"
```

Comment:
hmm what does "Single Doc for GCP" mean? sounds a bit confusing to me

Reply:
I should really be clear @sherry-x  I keep pushing my work in progress to this branch, so some of it is transitory and not for review. The above is definitely a placeholder text for myself. Originally I thought I could get a quick initial impression from you guys. Anyway, I will ask here when I am ready for you all to review this PR. Until then, please ignore.

Reply:
I see, it might be good to describe what type of review you're looking for now ;)